client/core/core.go

```diff
@@ -5244,9 +5298,12 @@ func (c *Core) prepareTrackedTrade(dc *dexConnection, form *TradeForm, crypter e
 		updated[toWallet.parent.AssetID] = struct{}{}
 	}
 
-	success = true
+	for assetID := range updated {
+		c.updateAssetBalance(assetID)
+	}
 
-	return corder, updated, nil
+	success = true
+	return corder, err
```

Comment:
`nil` instead of `err`?  If you don't know if `err` will be nil here, there should be an `if` statement to return differently depending on that.  It's ill-advised to return non-zero values when the error can be non-nil.

Reply:
yeah, should be `nil`.